src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -1598,6 +1598,16 @@ public Task Continue(DebuggedThread thread)
             return Execute(thread);
         }
 
+        public async Task Jump(string filename, int line)
+        {
+            await MICommandFactory.ExecJump(filename, line);
+        }
+
+        public async Task Jump(ulong address)
+        {
```

Comment:
>{ [](start = 8, length = 1)

As Andrew talked about, I believe we need to also:
- Take an exclusive lock to make sure that no other operation is happening
- Block until we have reentered break state.

Reply:
So maybe setting $pc would be better after all to avoid the jump command problems.

Reply:
It could be. I am not allowed to look at GDB code -- only documentation. So I don't know if the jump commands handle additional scenarios that would be good to support, or if they are essentially equivalent. If you have the cycles and the interest and want to take a look at the code and see if the jump commands do more, we would be happy to utilize your advice.

Reply:
Well the [jump cmd](https://github.com/bminor/binutils-gdb/blob/4258df85f13dbaf9a1cb5b523bf2b9d390b25e69/gdb/infcmd.c#L1126-L1137) is equal to `set $pc` + `continue` [it seems](https://github.com/bminor/binutils-gdb/blob/c651f0a6147e4de96326f6eecdff5520b9159816/gdb/infrun.c#L2898-L2922).